--- io.elementary.music/src/Objects/MediaKeyListener.vala	2018-02-12 15:43:24.398857124 -0200
+++ io.elementary.music/src/Objects/MediaKeyListener.vala	2018-02-12 15:44:00.698648913 -0200
@@ -51,7 +51,7 @@
         assert (media_object == null);
 
         try {
-            media_object = Bus.get_proxy_sync (BusType.SESSION, "org.gnome.SettingsDaemon", "/org/gnome/SettingsDaemon/MediaKeys");
+            media_object = Bus.get_proxy_sync (BusType.SESSION, "org.gnome.SettingsDaemon.MediaKeys", "/org/gnome/SettingsDaemon/MediaKeys");
         } catch (IOError e) {
             warning ("Mediakeys error: %s", e.message);
         }

