--- ../../../elementary/screenshot/src/Application.vala	2020-06-15 22:35:00.321450358 -0300
+++ ../../../elementary/screenshot/src/2Application.vala	2020-06-15 22:40:27.752228161 -0300
@@ -77,6 +77,17 @@
                 return;
             }
 
+            // set the elementary theme if the system use a diferent theme
+            var gtk_settings = new Gtk.Settings.get_default ();
+            if (gtk_settings.gtk_theme_name != "elementary") {
+                gtk_settings.gtk_theme_name = "elementary";
+            }
+
+            // set the elementary icon theme if the system use a diferent theme
+            if (gtk_settings.gtk_icon_theme_name != "elementary") {
+                gtk_settings.gtk_icon_theme_name = "elementary";
+            }
+
             window = new ScreenshotWindow ();
             window.get_style_context ().add_class ("rounded");
             window.set_application (this);
