--- a/src/PantheonCalculator.vala	2020-05-11 18:03:02.048751435 -0300
+++ b/src/PantheonCalculator.vala	2020-05-11 19:08:49.187711447 -0300
@@ -48,6 +48,17 @@
         }

         public override void activate () {
+            // set elementary theme if the system use a diferent theme
+            var gtk_settings = Gtk.Settings.get_default ();
+            if (gtk_settings.gtk_theme_name != "elementary") {
+                gtk_settings.gtk_theme_name = "elementary";
+            }
+
+            // set the elementary icon theme if the system use a diferent theme
+            if (gtk_settings.gtk_icon_theme_name != "elementary") {
+                gtk_settings.gtk_icon_theme_name = "elementary";
+            }
+
             window = new PantheonCalculator.MainWindow ();
             this.add_window (window);
         }
